installer/bundle/bundle_skel.sh

```diff
@@ -548,6 +520,7 @@ do
             echo "EXTRACT DIR:     $EXTRACT_DIR" >&2
             echo "SCRIPT:          $SCRIPT" >&2
             echo >&2
+            debugMode=true
```

Comment:
No, this was intended. Since I tried to pass the same arguments that the omsagent bundle was given, and pass to the SCX installer, I wanted to track whether --debug was used in the same way that forceFlag is used and tracked.